Source/qol/stash.cpp

```diff
@@ -423,7 +424,7 @@ uint16_t CheckStashHLight(Point mousePosition)
 
 	ClearPanel();
 
-	uint16_t itemId = abs(Stash.stashGrids[Stash.page][slot.x][slot.y]);
+	uint16_t itemId = abs(Stash.stashGrids[Stash.GetPage()][slot.x][slot.y]);

```

Comment:
This calls `abs` on a `uint16_t`. Redundant or a bug?

Reply:
Redundant, leftover from back when they where `int8_t`

Reply:
Will be fixed once this is merged: https://github.com/diasurgical/devilutionX/pull/4239/files#diff-fcb318e528e35c32e1bb412094800c059f6bea7a86003f15541e06bde8116fc0R415